README.md

```diff
@@ -1,2 +1,81 @@
 # Robot-Bug
 Srishti 2020
+## Abstract:<br/>
+<p align="justify">
+Aim of the project is to make a robot bug that can be used to collection information from places that are inaccessible for humans<br/></p>
+<p align="center">
+		
+![Robot Bug View 2](https://user-images.githubusercontent.com/55125159/88784215-66a72980-d1ad-11ea-9a5b-bbb6d0e23371.jpeg)
+
+## Motivation:<br/>
+<p align="justify">
+We have been hearing of robots being used mainly in factories for automating time-consuming processes and how they have radically changed the way work is being done. In the current times, the focus is on how the use of robotics outside the factory can change our lives over the coming decades. This is not just limited to easing out the day to day tasks but also includes the use of robots for more complex tasks. 
+
+## Mechanical Aspects of Design:
+<p align="justify">
+	<ol>
+<li>3 servo motors have been used for the project. The function of the servo motor placed along the side face of bug’s body rotates the bevel gears which is turn raise the arm up. <br/></li>
+<li>Once the arms are raised, the 2 servo motors being attached on the arms(1 on each) then moves the arms forward. <br/></li>
+<li>Hence first the arms are raised, then they move forward, touch down and then bot moves forward. <br/><li>
```

Comment:
Minor error. Change closing tag as </li> instead of <li>

Reply:
Correction has been made. Please review it.